host/sgx/windows/enter.asm

```diff
@@ -149,29 +171,19 @@ return_from_ecall:
     lfence
 
     ;; Set ARG3 (out)
-    mov rbx, ARG1OUT
+    mov r10, ARG1OUT
     mov rax, qword ptr [rbp+48]
-    mov qword ptr [rax], rbx
+    mov qword ptr [rax], r10
 
     ;; Set ARG4 (out)
-    mov rbx, ARG2OUT
+    mov r10, ARG2OUT
     mov rax, qword ptr [rbp+56]
-    mov qword ptr [rax], rbx
-
-    ;; Restore registers:
-    pop r15
-    pop r14
-    pop r13
-    pop r12
-    pop rsi
-    pop rdi
-    pop rbx
-
+    mov qword ptr [rax], r10
```

Comment:
Suggestion about pre-existing code.
Consider using a define instead of hard-coded offsets (e.g. rbp+48)

Reply:
Good idea, I've added definitions for these values for readability.